weave/weave_client.py

```diff
@@ -489,7 +489,8 @@ def finish_call(
             summary = sum_dict_leaves([child.summary or {} for child in call._children])
         elif isinstance(output, dict) and "usage" in output and "model" in output:
             summary["usage"] = {}
-            summary["usage"][output["model"]] = {"requests": 1, **output["usage"]}
+            model_key = str(output["model"])  # Convert ObjectRef to string
```

Comment:
Let's work on a different fix here: https://github.com/wandb/weave/pull/1891

Reply:
Reverted my changes to allow for the changes in #1891